src/middleware/src/storage/src/websql.js

```diff
@@ -7,20 +7,20 @@ import isFunction from 'lodash/isFunction';
 import isString from 'lodash/isString';
 const idAttribute = process.env.KINVEY_ID_ATTRIBUTE || '_id';
 const masterCollectionName = 'sqlite_master';
-const size = 5 * 1000 * 1000; // Database size in bytes
 let dbCache = {};
-let isSupported = undefined;
+let isSupported;
 
 export default class WebSQL {
-  constructor(name = 'kinvey') {
+  constructor(name = 'kinvey', size = 5000) {
     this.name = name;
+    this.size = size;
   }
 
   openDatabase() {
     let db = dbCache[this.name];
 
     if (!db) {
-      db = global.openDatabase(this.name, 1, '', size);
+      db = global.openDatabase(this.name, 1, '', this.size);
```

Comment:
Since it gets prompted to the end user, I'm a little unsure (the end user presumably doesn't know what Kinvey is). Maybe something like "Local Cache", which indicates the purpose of the database?